app/policies/proposal_policy.rb

```diff
@@ -98,7 +98,7 @@ def pending_delegate?
 
   def pending_approval!
     check(self.pending_approver? || self.pending_delegate?,
-          "A response has already been logged a response for this proposal")
+          "A response has already been logged for this proposal")
```

Comment:
Since a request goes through multiple approvals in the course of its lifetime, and thus multiple approvers, and since the complete approval history is already available in the /proposals/:id and email summary views, adding it to the error message could be both misleading and redundant.